monai/bundle/scripts.py

```diff
@@ -822,6 +822,56 @@ def verify_metadata(
     logger.info("metadata is verified with no error.")
 
 
+def _get_net_io_info(
+    parser: ConfigParser | None = None, prefix: str = "_meta_#network_data_format", p: int = 1, n: int = 1, any: int = 1
+) -> list[Any]:
+    """
+    Get the input and output information defined in the metadata.
+
+    Args:
+        parser: a parser of the given bundle.
+        prefix: a prefix for the input and output keys, which will be combined as `prefix#input` and
+            `prefix#output` to parse the input and output information in the `metadata.json` file of
+            bundle, default to `meta_#network_data_format`.
+        p: power factor to generate input data shape if dim of expected shape is "x**p", default to 1.
+        n: multiply factor to generate input data shape if dim of expected shape is "x*n", default to 1.
+        any: specified size to generate input data shape if dim of expected shape is "*", default to 1.
+
+    Return:
+        a sequence contains the input and output infomation.
+    """
+    if not isinstance(parser, ConfigParser):
+        raise AttributeError(f"'parser' should be a ConfigParser, got {type(parser)}.")
+
+    inputs_shape = []
+    try:
+        prefix_key = f"{prefix}#inputs"
+        inputs_info_dict = parser[prefix_key]
```

Comment:
I don't quite understand what's this `inputs_info_dict` mean here?

Thanks.

Reply:
This parameter is trying to parse all the parameters in `prefix#inputs` in case there are keys in addition to `image` llike the mednist gan with `latent`.